README.md

```diff
@@ -1,173 +1,40 @@
-```
-
-If you want to install for python3, use the following packages instead:
-```
-apt install python3-dateutil python3-polib python3-psycopg2 python3-shapely python3-regex python3-requests
-```
-
-### Alt: python dependencies in a virtualenv
-
-Alternatively instal python-virtualenv and create a new virtualenv.
-
-Setup system dependencies (Debian Stretch)
-```
-apt install build-essential python-dev python-virtualenv libpq-dev protobuf-compiler libprotobuf-dev
-```
-
-Create a python virtualenv, active it and install python dependencies
-```
-virtualenv --python=python2.7 osmose-backend-venv
-source osmose-backend-venv/bin/activate
-pip install -r requirements.txt
-```
-
-To run tests, additional packages are needed.
-```
-pip install -r requirements-dev.txt
-```
-
-### Compile the OMS PBF parser
-
-Build the native python module lib to parse .osm.pbf files:
-```
-apt install g++ libboost-python-dev libosmpbf-dev make pkg-config
-
-cd modules/osm_pbf_parser/
-make
-```
+# Backend part of Osmose QA tool
 
+![Build Status](https://api.travis-ci.com/osm-fr/osmose-backend.svg?branch=master)
 
-Installation Database
----------------------
-
-Setup system dependencies (Debian Stretch)
-```
-apt install postgresql-9.6 postgresql-contrib-9.6 postgresql-9.6-postgis-2.1
-```
-
-As postgres user:
-```
-createuser osmose
-# Set your own password
-psql -c "ALTER ROLE osmose WITH PASSWORD '-osmose-';"
-createdb -E UTF8 -T template0 -O osmose osmose
-# Enable extensions
-psql -c "CREATE extension hstore; CREATE extension fuzzystrmatch; CREATE extension unaccent; CREATE extension postgis;" osmose
-psql -c "GRANT SELECT,UPDATE,DELETE ON TABLE spatial_ref_sys TO osmose;" osmose
-psql -c "GRANT SELECT,UPDATE,DELETE,INSERT ON TABLE geometry_columns TO osmose;" osmose
-```
-
-
-Dependencies
-------------
-
-Java JRE for osmosis (Ubuntu Server 14.04):
-```
-apt install openjdk-7-jre-headless
-```
-
-osmosis is installed in osmosis/osmosis-0.47/.
-osmconvert is installed in osmconvert/.
-
-
-Configuration
--------------
-A few paths are hardcoded in modules/config.py, and should be adapted or created.
-
-  - dir_osmose is the path of where osmose is installed
-  - dir_work is where extracts are stored, and results generated.
-  - url_frontend_update is the url used to send results generated by analyses
-
-
-The local postgresql database should be configured in osmose_config.py:
+This is the part of [Osmose](http://osmose.openstreetmap.fr) which analyses OSM
+and send results to frontend. This works as following:
 
-  - db_base = osmose # database name
-  - db_user = osmose # database user
-  - db_password = # database password if needed
-  - db_host = # database hostname if needed
+  - an .osm.pbf extract is downloaded
+  - analyses are run directly on .osm.pbf file, or on the database
+  - analyses resultat are uploaded to the frontend
+  - by default, temporary extract files and database are purged
```

Comment:
It is not completely true - .osm.pbf file is preserved, and used on next run to use diffs instead of redownloading file.